packages/strapi-admin/admin/src/components/Users/LoginModalSection/index.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ModalSection } from 'strapi-helper-plugin';
+import { Padded } from '@buffetjs/core';
+import { Row } from 'reactstrap';
+import { useIntl } from 'react-intl';
+import loginSettingsForm from 'ee_else_ce/components/Users/ModalCreateBody/utils/loginSettingsForm';
+
+import Input from '../../SizedInput';
+import Wrapper from '../ModalCreateBody/Wrapper';
+
+const LoginModalSection = ({ isDisabled, modifiedData, onChange, formErrors }) => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <ModalSection>
+      <Wrapper>
+        <Padded top size="smd">
+          <Row>
+            {Object.keys(loginSettingsForm).map(inputName => {
+              if (loginSettingsForm[inputName].Component) {
+                const { Component } = loginSettingsForm[inputName];
+
+                return (
+                  <Component
+                    key={inputName}
+                    value={modifiedData[inputName]}
```

Comment:
Could you please extract this in a named variable please? I think it would be easier to understand since there's something with:

- `modifiedData[inputName]`
- `loginSettingsForm[inputName]`

Spotting the difference between these two might be complex (in fact, it is for me 😅 )